sc/social/like/browser/imagefallback.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.formwidget.namedfile.converter import b64decode_file
+from plone.namedfile.browser import Download
+from plone.namedfile.file import NamedImage
+from sc.social.like.interfaces import ISocialLikeSettings
+
+
+class ImageFallBack(Download):
+
+    filename = None
+    data = None
+
+    def __call__(self):
+        self.setup()
+        super(ImageFallBack, self).__call__()
+
+    def setup(self):
+        fallback_image = api.portal.get_registry_record('fallback_image', interface=ISocialLikeSettings)
```

Comment:
you forgot the default value; without it it will fail if the upgrade step has not been run.

Reply:
Ok